source/_posts/2025-03-05-release-20253.markdown

```diff
@@ -0,0 +1,563 @@
+- [Tile cards](#tile-cards)
+  - [Knowing which tile card can be controlled](#knowing-which-tile-card-can-be-controlled)
+  - [Position tile card features](#position-tile-card-features)
+  - [New tile card features](#new-tile-card-features)
+    - [Switch toggle](#switch-toggle)
+    - [Counter actions](#counter-actions)
+  - [Tiny interaction improvements](#tiny-interaction-improvements)
+  - [Editor improvements](#editor-improvements)
+- [Assist chat now has streaming responses](#assist-chat-now-has-streaming-responses)
+- [Integrations](#integrations)
+  - [New integrations](#new-integrations)
+  - [Noteworthy improvements to existing integrations](#noteworthy-improvements-to-existing-integrations)
+- [Other noteworthy changes](#other-noteworthy-changes)
+- [Iterating on the legends of graphs](#iterating-on-the-legends-of-graphs)
+- [Grouping / clustering of trackables on the maps](#grouping--clustering-of-trackables-on-the-maps)
+- [Need help? Join the community!](#need-help-join-the-community)
+- [Backward-incompatible changes](#backward-incompatible-changes)
+- [All changes](#all-changes)
+
+## Dashboard view headers
+
+This release brings the ability to add headers to your dashboards. This allows
+you to add a title and a welcoming text (using Markdown and even templates)
+to your dashboards. Additionally, badges can be added to the header.
+
+The nice thing about the header is that it can be aligned in different ways.
+It comes with a responsive layout by default, but you can also set it to
+always left align or always center align.
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-view-header.png" />
+
+As for the badges, they can be positioned as well, either below the text
+(default) or above the text.
+
+## Tile cards
+
+A lot of touches and improvements have been made to the tile cards in this
+release. The tile card is one of the most versatile cards in Home Assistant,
+and we have made it even better!
+
+### Knowing which tile card can be controlled
+
+Tile cards are awesome! It is arguably the best and most versatile dashboard
+card Home Assistant current has. But ever since its creation, it had a little
+flaw that you might have ran into yourself a few time...
+
+How would you know if tapping a tile card would trigger an action or when
+it would just display the current state value of a sensor? We have subtly fixed
+these things in this release!
+
+<img class="no-shadow" alt="TODO: Replace." src="/images/blog/2025-03/temp-tile-card-shapes.png" />
+
+When there is no action (or only a more info attached), we don't show the circle
+when it does have an actions, it will show the circle, meaning you can tap that
+little button.
+
+### Position tile card features
+
+Ever since we added support for resizing cards, we gained the ability to make
+very wide tile cards. These cards would have a lot of empty space surrounding
+it, which, well at times could be helpful I guess.
```

Comment:
```suggestion
Ever since we added support for resizing cards, we gained the ability to make
very wide tile cards. These cards would have a lot of empty space surrounding
them, which, well, at times could be helpful, I guess.
```